<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sysOpt</class>
 <widget class="QWidget" name="sysOpt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>270</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>SYSTEM MENU</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnminimumwidth="160,160">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="2" column="1" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pb_device">
       <property name="minimumSize">
        <size>
         <width>124</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>124</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>DEVICE</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pb_camera">
       <property name="minimumSize">
        <size>
         <width>124</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>124</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CAMERA</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pb_video">
       <property name="minimumSize">
        <size>
         <width>124</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>124</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>VIDEO</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pb_ethernet">
       <property name="minimumSize">
        <size>
         <width>124</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>124</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ETHERNET</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pb_wifi">
       <property name="minimumSize">
        <size>
         <width>124</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>124</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wi-Fi</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pb_blueTooth">
       <property name="minimumSize">
        <size>
         <width>124</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>124</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>BLUETOOTH</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder></zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <tabstops>
  <tabstop>pb_camera</tabstop>
  <tabstop>pb_video</tabstop>
  <tabstop>pb_wifi</tabstop>
  <tabstop>pb_blueTooth</tabstop>
  <tabstop>pb_ethernet</tabstop>
  <tabstop>pb_device</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
